etcdmgmt/etcd-mgmt.go

```diff
@@ -35,6 +36,11 @@ func StartEtcd() error {
 		"-initial-advertise-peer-urls", initialAdvPeerUrls,
 		"--initial-cluster", "default="+listenPeerUrls)
 
+	// Don't kill chlid process (etcd) upon ^C (SIGINT) of main glusterd process
```

Comment:
I am honestly failing to understand your arguments here.  

Point 1 : Currently glusterd2 daemon itself is not daemonized and hence once you execute it it doesnt return back to the prompt and with that you bring in interrupts, however this wouldn't be true as we have to daemonize it (this is how existing glusterd also behave). So if your observation is that etcd doesnt go down in SIGKILL or SIGTERM we dont need this change at all as we can run the current glusterd2 instance in background to avoid the command prompt to get stuck.

Point 2 : Being a programmer I expect these basic logics to be thought out before sending the changes. A github issue will highlight what is the problem we are tryung to solve but it will *never* say how are you going to achieve it. Starting etcd instance without checking whether an already etcd process is running or not is a *mistake*, no matter whatever reason you provide and I dont want to get into that further.

Point 3: Word 'daemonize' has some significant meaning. Daemonizing a process doesn't only mean changing the a single attribute of a process.  Again to repeat daemonize itself explains what needs to be done.